games-engines/gemrb/additional-files/gemrb.rdef

```diff
@@ -0,0 +1,36 @@
+resource app_version {
+       major  = 0,
+       middle = 9,
+       minor  = 3,
```

Comment:
Usually, these are set as "variables" which are replaced with sed in the INSTALL function with values derived from `$portVersion`, and therefore this file has a `.rdef.in` file extension. This is so that it doesn't need to be updated manually for every new version.

You'll find many examples in HaikuPorts how to do this.